content/daytrip/eu/gb/78-derngate.md

```diff
@@ -3,7 +3,7 @@ slug: "daytrip/eu/gb/78-derngate"
 date: '2001-01-30T04:37:00'
 lat: '52.202016'
 lng: '-0.894286'
-location: Tarragon Way, Blacky More, East Hunsbury, Wootton, West Northamptonshire, England, NN4 0SF, United Kingdom
+location: Tarragon Way, Blacky More, East Hunsbury, Wootton, West Northamptonshire, NN4 0SF, United Kingdom
```

Comment:
This is clearly wrong. The address should be 78 Derngate, Northampton. The address given here is several miles away in a residential area.